client_handler.cpp

```diff
@@ -30,6 +30,11 @@
 #include "sbcommon.h"
 #include "DlgCertification.h"

```

Comment:
Can we remove this `include`?

Reply:
Thanks, I've removed it.

(NOTE: Keep DlgCertification implemention itself to refer as a custom dialog for a while.)

Reply:
I've also rebased with master.